Ha foreman

discreet opened the pull request:
Guest blog post for HA Foreman implementation.

Commit:
initial draft Journey to HA

Commit:
gues blog post on high availability

GregSutcliffe commented:
I don't think you meant to commit two separate .md files there @discreet :) Could you remove whichever is the erroneous one.

Commit:
removed old commit file

discreet commented:
@GregSutcliffe my apologies, erroneous .md file should be removed now.

GregSutcliffe commented on _posts/2015-12-03-journey_to_high_availability.md:
I think you  mean `smart proxy` rather than `Foreman node` right? You don't need a generic certificate for every single host...

GregSutcliffe commented on _posts/2015-12-03-journey_to_high_availability.md:
grammar issue here - "which we filed _an_ issue for<comma> which is tracked as"

GregSutcliffe commented on _posts/2015-12-03-journey_to_high_availability.md:
I'd consider putting this higher up (maybe just after the requirements list?) - it's a bit of a wall-o-text otehrwise and an image in the middle will break it up visually, as well as give people something to refer to as they read each section below.

GregSutcliffe commented:
@discreet Looks pretty good, justa few niggles - thanks! Do you want to add a link to the Case Study? Also, feel free to squash the commits (or I can do it on merge if you like).

discreet commented on _posts/2015-12-03-journey_to_high_availability.md:
@GregSutcliffe we generated a certificate (abcforeman.domain.com) to be distributed to each of our foreman hosts for web traffic.  i guess this isn't 100% necessary if you make use of alt dns names though on the hostname based certificates.

discreet commented:
@GregSutcliffe Thanks. I moved up the diagram and added a link to the case study in the Overview. I fixed the grammar issue as well. I left a comment on the part about the certs, but it is currently unchanged right now. I'll push the changes I made thus far.

Commit:
grammar changes and formatting

GregSutcliffe commented on _posts/2015-12-03-journey_to_high_availability.md:
Right, thats for outbound traffic to the Proxies etc, yes? I just want to avoid confusion, as its written I could interpret that as _every_ host in Foreman, which I don't think is right :)

GregSutcliffe commented on _posts/2015-12-03-journey_to_high_availability.md:
Maybe just preface this with a line of explanation? "If you want to see a live discussion of this topic, here's an interview as part of the Foreman Case Study series" or similar?

GregSutcliffe commented:
@discreet looking pretty sweet, thats the last of my comments I think :P

Commit:
added line for Foreman Case Study video and clarification for certificates

discreet commented:
@GregSutcliffe Thanks, added those two things. Hopefully the certificates part reads more clear now.

GregSutcliffe commented:
Merged as b93a7f6, thanks @discreet !
